apps/examples/elf/Kconfig

```diff
@@ -0,0 +1,23 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_ELF
```

Comment:
This looks loader example for ELF type. I don't know `EXAMPLES_ELF` is appropriate.

Reply:
This example only build binary of ELF format